src/app/visual/date-range-picker/date-range-picker-visual.component.html

```diff
@@ -0,0 +1,23 @@
+<h2>Date range picker</h2>
+<form
+  #form="ngForm"
+>
```

Comment:
Probably would be more useful to write this using Reactive Forms, since the demo would be written that way.

Reply:
I actually already have a reactive form demo here too. Should I remove the templated one?

Reply:
Oh, must have missed that. Nope, let's have both :)